packages/animations/test/open_container_test.dart

```diff
@@ -1783,47 +1784,6 @@ void main() {
       expect(modalRoute.settings, routeSettings);
     },
   );
-
-  // Regression test for https://github.com/flutter/flutter/issues/72238.
-  testWidgets(
```

Comment:
This test was always grabbing the wrong opacity and never actually tested the issue in question